src/js/claims-status/utils/appeals-v2-helpers.jsx

```diff
@@ -43,6 +52,55 @@ export const ALERT_TYPES = {
   droHearingNoShow: 'dro_hearing_no_show',
 };
 
+/**
+ * Takes an array of appeals and returns another array of issue descriptions
+ * and where in the appeal lifecycle each issue is (open, remand, granted, denied)
+ * @typedef {Object} issue an individual issue - many issues can be a part of a single appeal
+ * @property {bool} active indicates whether an appeal is open or closed
+ * @property {string} description more info about the specific injury in the issue
+ * @property {string} diagnosticCode a codified version of the description
+ * @property {('field_grant'|'withdrawn'|'allowed'|'denied'|'remand'|'cavc_remand')} lastAction
+ * @property {string} date TO-DO: unsure of what this date siginifies
+ * ------------------------------------------------------------------------------------------------
+ * @typedef {Object} segmentedIssue issue with descriptor and status information
+ * @property {('granted'|'remand'|'allowed'|'denied')} status lifecycle stage of an issue
+ * @property {string} description pass-through for the description info of passed in issue object
+ * ------------------------------------------------------------------------------------------------
+ * @param {issue[]} issues all the individual issues that are attached to an appeal
+ * @returns {segmentedIssue[]} an array of issue objects with stuses and descriptions
+ */
+export function categorizeIssues(issues) {
```

Comment:
This kind of seems like it would be returning all the issues whole, but categorized like
```js
{
  granted: [ ... ],
  denied: [ ... ],
  withdrawn: [ ... ]
}
```

The implementation you've got is good, but I'm wondering if this could use a rename.

Reply:
Yeah, fair point. And maybe that _should_ be the implementation, but it feels ok to keep it as a lean mapping of issue statuses to display statuses.

As for the name, how about `addStatusToIssue()`?

Reply:
Naming is hard. Turns out I can't think of a better one. :man_shrugging: